vizro-core/src/vizro/models/_controls/filter.py

```diff
@@ -15,19 +15,32 @@
 from vizro.managers import data_manager, model_manager
 from vizro.managers._model_manager import ModelID
 from vizro.models import Action, VizroBaseModel
-from vizro.models._components.form import Checklist, Dropdown, RadioItems, RangeSlider, Slider
+from vizro.models._components.form import (
+    Checklist,
+    DatePicker,
+    Dropdown,
+    RadioItems,
+    RangeSlider,
+    Slider,
+)
 from vizro.models._models_utils import _log_call
 from vizro.models.types import MultiValueType, SelectorType
 
-# TODO: Add temporal when relevant component is available
-SELECTOR_DEFAULTS = {"numerical": RangeSlider, "categorical": Dropdown}
+SELECTOR_DEFAULTS = {"numerical": RangeSlider, "categorical": Dropdown, "temporal": DatePicker}
 
 # Ideally we might define these as NumericalSelectorType = Union[RangeSlider, Slider] etc., but that will not work
 # with isinstance checks.
-SELECTORS = {"numerical": (RangeSlider, Slider), "categorical": (Checklist, Dropdown, RadioItems)}
+SELECTORS = {
+    "numerical": (RangeSlider, Slider),
+    "categorical": (Checklist, Dropdown, RadioItems),
+    "temporal": DatePicker,
+}
```

Comment:
Maybe we can actually combine these two mappings?

Reply:
You mean `SELECTORS` and `SELECTOR_DEFAULTS`? If so then I agree - we just say that the first entry in each tuple is the default selector for that key. The only place that `SELECTOR_DEFAULTS` is used can be removed and changed to `SELECTORS[self._column_type][0]`.

The catch here is that it's fragile because someone could change the order of e.g. `RangeSlider` and `Slider` without realising that it's changing the default selector. So if we make this change we need a comment right above `SELECTORS` explaining clearly that the first entry in each tuple is the default selector.

Note you'll need to swap the order of `categorical` to make `Dropdown` the first also. Hopefully if you don't do that then our tests will fail and tell us. If they don't then actually our tests aren't robust enough to catch this and we shouldn't make the change here at all.

Reply:
Yes, that was what I was thinking about :) I think a comment should suffice because we rarely does this mapping